src/core/announcements/index.ts

```diff
@@ -17,5 +17,7 @@
  *   - `import { foo } from "@dsnp/sdk/contracts"; foo.functionInFoo();`
  */
 
-export * from "./messages";
-export * from "./messageErrors";
+export * from "./errors";
+export * from "./announcementTypes";
```

Comment:
Given that we changed `messageErrors` to `errors`, I wonder if `announcementTypes` should be just `types` here.

Reply:
Ended up with factories in out of band conversation